src/net_processing.cpp

```diff
@@ -724,20 +712,17 @@ bool ProcessMessage(CNode *pfrom, std::string strCommand, CDataStream &vRecv, in
 
         pfrom->ReadConfigFromXVersion();
 
-        pfrom->PushMessage(NetMsgType::XVERACK);
-        pfrom->state_incoming = ConnectionStateIncoming::READY;
-        handleAddressAfterInit(pfrom);
-        enableSendHeaders(pfrom);
-        enableCompactBlocks(pfrom);
+        pfrom->PushMessage(NetMsgType::XVERACK_OLD);
+        // handleAddressAfterInit(pfrom);
+        // enableSendHeaders(pfrom);
+        // enableCompactBlocks(pfrom);
```

Comment:
several lines of code commented out? don't we still need them?

Reply:
they were moved. they are still called under the verack message